src/main/java/lottomanual/domain/Lotto.java

```diff
@@ -0,0 +1,83 @@
+package lottomanual.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+	public static final int NUMBERS_SIZE_LIMIT = 6;
+
+	private final Set<LottoNumber> numbers;
+
+	public Lotto(String numbersText) {
+		this(toSet(toList(numbersText)));
+	}
+
+	public Lotto(List<Integer> numbers) {
+		this(toSet(numbers));
```

Comment:
```suggestion
		this(new TreeSet(numbers));
```
아마 이렇게 표현해도 될 것 같아요!